controllers/cnfappmac_controller.go

```diff
@@ -229,6 +218,29 @@ func (r *CNFAppMacReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	return ctrl.Result{}, nil
 }
 
+func (r *CNFAppMacReconciler) getResMap(resName, podName, namespace, nwName string) (map[string]interface{}, error) {
+	resName = strings.ReplaceAll(resName, "/", "_")
+	resName = strings.ReplaceAll(resName, "-", "_")
+	resName = strings.ReplaceAll(resName, ".", "_")
+	resName = strings.ToUpper(resName)
+	envName := "PCIDEVICE_" + resName
+
+	pciValue, err := getContainerEnvValue(podName, namespace, envName)
+	if err != nil {
+		return nil, err
+	}
+	pciValue = strings.TrimSuffix(pciValue, "\n")
+	pciValue = strings.TrimSuffix(pciValue, "\r")
+	fmt.Println(pciValue)
```

Comment:
Should this be `log.Info()` instead?

```suggestion
	log.Info("PCI value", pciValue)
```

Reply:
Done.